__mocks__/react-native-permissions.ts

```diff
@@ -0,0 +1,77 @@
+import {PERMISSIONS, RESULTS} from 'react-native-permissions/dist/commonjs/permissions';
```

Comment:
Would that work?
```suggestion
import {PERMISSIONS, RESULTS} from 'react-native-permissions';
```

Reply:
No, importing these from react-native-permissions directly are breaking the unit tests, this is the reason why I've added this module declaration.
I've tried that on this commit: https://github.com/Expensify/App/pull/35886/commits/1fdb8fdffafbd3be495c43ecdd4b83a486be9d99 but had to revert it